content/docs/about/overview.md

```diff
@@ -91,6 +91,7 @@ The kgateway project provides the following extensions on top of the {{< reuse "
   {{< card link="/docs/integrations/" title="Integrations" tag="Setup" >}}
   {{< card link="/docs/resiliency/mirroring/" title="Mirroring" tag="Resiliency" >}}
   {{< card link="/docs/traffic-management/transformations/" title="Transformations" tag="Traffic" >}}
+  {{< card link="/docs/traffic-management/weighted-routes/" title="Weighted routes" tag="Traffic" >}}
```

Comment:
I added this to the about page under "[extensions](https://kgateway.dev/docs/about/overview/#extensions)," but maybe there is a better place to put this in the about pages?